app/src/main/java/com/truongdc21/quickquotes/Presenter/FragmentHome/HomeFragmentContact.kt

```diff
@@ -1,4 +1,27 @@
 package com.truongdc21.quickquotes.presenter.fragmentHome
 
+import com.truongdc21.quickquotes.data.model.Quotes
+import kotlin.Exception
+
 interface HomeFragmentContact {
+
+    interface Presentet {
```

Comment:
```suggestion
    interface Presenter : BasePresenter<View> {
```

Em tham khảo project mẫu của anh làm ở đây:
https://github.com/daolq-2712/Structure_Android/blob/mvp_kotlin/app/src/main/java/com/sun/mvp/screen/listmovie/MoviesContract.kt

Reply:
@daolq-2712 là phải truyền View qua bên presenter rồi set View tront presenter luôn hả a ?

Reply:
@truongdc21 
setup relative giữa view & presenter e và mối quan hệ này là 1-1

Reply:
@daolq-2712 Dạ anh